sim/hunter/items.go

```diff
@@ -643,3 +643,50 @@ func (hunter *Hunter) newBloodlashProcItem(bonusStrength float64, spellID int32)
 
 	hunter.ItemSwap.RegisterProc(BloodlashBow, aura)
 }
+
+// Striking a higher level enemy applies a stack of Coup, increasing their damage taken from your next Kill Shot by 5% per stack, stacking up to 20 times.
+func (hunter *Hunter) ApplyRegicideHunterEffect(aura *core.Aura) {
+	// Coup debuff array
+	debuffAuras := hunter.NewEnemyAuraArray(func(unit *core.Unit, _ int32) *core.Aura {
+		aura := unit.RegisterAura(core.Aura{
+			ActionID:  core.ActionID{SpellID: 1231417},
```

Comment:
Here's the debuff used for hunter https://www.wowhead.com/classic-ptr/spell=1231765/coup also a 15s duration